src/app/shared/components/template/components/layout/popup.ts

```diff
@@ -83,25 +88,19 @@ import { TemplateContainerComponent } from "../../template-container.component";
  * When opening a template as a popup, provide a minimal interface and load
  * the template directly as a regular template-container element
  */
-export class TemplatePopupComponent implements ITemplateContainerProps, OnInit {
-  @Input() name: string;
-  @Input() templatename: string;
-  @Input() parent?: TemplateContainerComponent;
-  @Input() row?: FlowTypes.TemplateRow;
-  /** Standalone mode shows in fullscreen and dismisses on completed/uncompleted emit events */
-  @Input() standalone?: boolean;
+export class TemplatePopupComponent {
+  @Input() props: ITemplatePopupComponentProps;
 
   constructor(private modalCtrl: ModalController) {}
 
-  ngOnInit() {}
-
   /**
    * When templates emit completed/uncompleted value from standalone popup close the popup
    * NOTE - we do not want to respond to non-standalone templates as this is done through template nav-actions
    * */
-  handleEmittedValue(value: string) {
-    if (this.standalone) {
-      if (["completed", "uncompleted"].includes(value)) {
+  handleEmittedValue(value: { emit_value: string; emit_data: any }) {
```

Comment:
previously standalone popups only emitted their emit_value (e.g. completed/uncompleted) on dismiss. To support the feedback/form implementation support added to also include optional data on emit (such as form data)